tlslistener/clienthelloconn.go

```diff
@@ -205,16 +205,16 @@ func (rrc *clientHelloRecordingConn) processHello(info *tls.ClientHelloInfo) (*t
 	// pre-defined tickets. If it doesn't we should again return some sort of error or just
 	// close the connection.
 	if !helloMsg.TicketSupported {
-		return rrc.helloError("ClientHello does not support session tickets", true)
+		return rrc.helloError("ClientHello does not support session tickets", false)
```

Comment:
Since `continueOnError` is always set to false now, let's just remove that parameter from `clientHelloRecordingConn.helloError`.  Sorry, I think that's different than what I originally told you!